apps/block_scout_web/config/config.exs

```diff
@@ -138,14 +133,14 @@ config :block_scout_web, BlockScoutWeb.SocialMedia,
 
 # Configures History
 price_chart_config =
-  if System.get_env("SHOW_PRICE_CHART", "true") != "false" do
+  if System.get_env("SHOW_PRICE_CHART", "false") != "false" do
```

Comment:
I believe we will need to set this if we want to show the coin price history chart, as it's now `false` by default